types/url-parse/url-parse-tests.ts

```diff
@@ -1,18 +1,16 @@
-import parse = require("url-parse");
+import URL = require("url-parse");
 
-const url1 = new URL("foo/bar", "https://github.com/");
-const url2 = parse("https://github.com/foo/bar?baz=true");
```

Comment:
calling parse as a funciton seems to be allowed in the documentation. to do that the exported value should have a type like:

```ts
declare const parse: {
    new  (address: string, location?: string | object, parser?: boolean | QueryParser): URLParse;
    (address: string, location?: string | object, parser?: boolean | QueryParser): URLParse;
};
export = parse;
```